src/js/transforms/DropZone.js

```diff
@@ -19,10 +37,12 @@ function DropZone(graph) {
     .dependency(Deps.SIGNALS, [sg.CELL, sg.MOUSE]);
 }
 
-var prototype = (DropZone.prototype = Object.create(Transform.prototype));
-prototype.constructor = DropZone;
-
-prototype.transform = function(input) {
+/**
+ * transform dropzone
+ * @param {string} input - todo.
+ * @return {object} output - todo
+ */
+DropZone.prototype.transform = function(input) {
```

Comment:
@arvind this may have been discussed but we prefer `ConstructorFn.prototype.fnName = ` to using an intermediate variable for obviousness and to avoid having to manually specify as much information via JSDoc (the more we can let JSDoc's "magic" do its thing, the easier it is to maintain the documentation)